projects/plugins/jetpack/locales.php

```diff
@@ -1,4 +1,8 @@
-<?php
+<?php // phpcs:disable
+/**
+ * This file is copied over from GlotPress/GlotPress-WP project and does not nessitate linting.
+ * @deprecated Please use 'projects/packages/compat/lib/locales.php' instead.
```

Comment:
Instead of duplicating the whole thing, this stub here could just `require_once` the new file from the package.

Reply:
Done! Just needed a little song and dance to make the `Squiz.Commenting.FileComment.Missing` linting rule happy.